CHANGELOG.md

```diff
@@ -1,11 +1,63 @@
+* [pr#4439](https://github.com/chef/chef/pull/4439)  Sometimes chocolately doesn't appear on the path
+* [pr#4432](https://github.com/chef/chef/pull/4432)  add get_rest etc calls to ServerAPI
+* [pr#4435](https://github.com/chef/chef/pull/4435)  add nokogiri to omnibus-chef
+* [pr#4433](https://github.com/chef/chef/pull/4433)  add a comment on exception triggers
+* [pr#4419](https://github.com/chef/chef/pull/4419)  explicitly adding .bat to service executable called by service in cae users remove .bat from PATHEXT
+* [pr#4426](https://github.com/chef/chef/pull/4426)  let kitchen-appbundle-updater float
+* [pr#4413](https://github.com/chef/chef/pull/4413)  configure chef client windows service to the correct chef directory
+* [pr#4377](https://github.com/chef/chef/pull/4377)  fixing candidate filtering and adding functional tests for chocolatey_package
+* [pr#4406](https://github.com/chef/chef/pull/4406)  Updating to the latest release of net-ssh to consume https://github.com/net-ssh/net-ssh/pull/280
+* [pr#4405](https://github.com/chef/chef/pull/4405)  ServerAPI will return a raw hash, so do that
+* [pr#4400](https://github.com/chef/chef/pull/4400)  inflate an environment after loading it
+* [pr#4387](https://github.com/chef/chef/pull/4387)  Initial 'trivial' acceptance test suite
+* [pr#4396](https://github.com/chef/chef/pull/4396)  Remove duplicate initialization of @password in user_v1
+* [pr#4344](https://github.com/chef/chef/pull/4344)  Warn (v. info) when reloading resources
+* [pr#4369](https://github.com/chef/chef/pull/4369)  Migrate omnibus-chef project/software definitions for chef in here
+* [pr#4106](https://github.com/chef/chef/pull/4106)  add chocolatey_package to core chef
+* [pr#4340](https://github.com/chef/chef/pull/4340)  Re-enable AIX mount functional test
+* [pr#4347](https://github.com/chef/chef/pull/4347)  add knife-windows to external specs
+* [pr#4321](https://github.com/chef/chef/pull/4321)  fix run_as_user of windows_service
+* [pr#4333](https://github.com/chef/chef/pull/4333)  no longer wait on node search to refresh vault but pass created ApiCient instead
+* [pr#4327](https://github.com/chef/chef/pull/4327)  tiny_server should retry Errno::EHOSTUNREACH
+* [pr#4325](https://github.com/chef/chef/pull/4325)  Pin win32-eventlog to 0.6.3 to avoid clashing CreateEvent definition
+* [pr#4312](https://github.com/chef/chef/pull/4312)  Updates the template to use omnitruck-direct.chef.io
+* [pr#4277](https://github.com/chef/chef/pull/4277)  non msi packages must explicitly provide a source attribute on install
+* [pr#4309](https://github.com/chef/chef/pull/4309)  tags always an array; fix set_unless
+* [pr#4278](https://github.com/chef/chef/pull/4278)  make file resource use properties
+* [pr#4288](https://github.com/chef/chef/pull/4288)  Fix no_proxy setting in chef-config
+* [pr#4286](https://github.com/chef/chef/pull/4286)  bump up timeout for travis
```

Comment:
#4286 #4327 #4347 #4340 #4387 #4426 #4433 #4371 #4436 #4445 #4461 #4466 #4386 are all internal testing fixes or not user-visible code changes.  debatable on a couple of them, but most of them are just noise in the CHANGELOG when consumed by external users.